modules/planning/tasks/deciders/speed_bounds_decider/st_boundary_mapper.cc

```diff
@@ -198,6 +199,15 @@ bool STBoundaryMapper::GetOverlapBoundaryPoints(
     return false;
   }
 
+  const auto* planning_status = PlanningContext::Instance()
+                                    ->mutable_planning_status()
+                                    ->mutable_change_lane();
+
+  double l_buffer =
+      planning_status->status() == ChangeLaneStatus::IN_CHANGE_LANE
+          ? FLAGS_lane_change_obstacle_nudge_l_buffer
+          : FLAGS_nonstatic_obstacle_nudge_l_buffer;
```

Comment:
do not use the gflag for nonstatic_obstacle_nudge_l_buffer just because they have the same value. 
that may introduce bugs when we change that gflag w/o being aware of the implact on change lane case. 
use another gflag or (existing gflag - a adjustment value).